packages/docs/src/examples/combobox/comboboxMultiple.vue

```diff
@@ -26,10 +26,7 @@
           multiple
           chips
         >
-          <template
-            slot="selection"
-            slot-scope="data"
-          >
+          <template slot:selection="data">
```

Comment:
this is throwing errors 
![image](https://user-images.githubusercontent.com/9064066/52865981-59a29180-3103-11e9-8356-51bde72d6ce5.png)